packages/mobile/src/screens/buy-sell-screen/BuySellModalScreen.tsx

```diff
@@ -0,0 +1,44 @@
+import { BottomSheetModalProvider } from '@gorhom/bottom-sheet'
+import { useRoute } from '@react-navigation/native'
+import { createNativeStackNavigator } from '@react-navigation/native-stack'
+
+import { ModalScreen } from 'app/components/core'
+
+import { useAppScreenOptions } from '../app-screen/useAppScreenOptions'
+
+import { BuySellScreen } from './BuySellScreen'
+import { ConfirmSwapScreen } from './ConfirmSwapScreen'
+import { TransactionResultScreen } from './TransactionResultScreen'
+
+const Stack = createNativeStackNavigator()
+
+const screenOptionOverrides = { headerRight: () => null }
+
+export const BuySellModalScreen = () => {
+  const screenOptions = useAppScreenOptions(screenOptionOverrides)
+  const { params } = useRoute()
+
+  return (
+    <ModalScreen>
+      <BottomSheetModalProvider>
+        <Stack.Navigator screenOptions={screenOptions}>
+          <Stack.Screen
+            name='BuySell'
```

Comment:
Oh jk just moved over here, excellent. Can you share this context with Reed because I think his flow may benefit from this too